docs/get-started/on-device-training.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
for the file name, is that better we call it training-on-device.md?

Reply:
I think `on-device-training.md` might be better considering this is the marketing name we want to go with.

Any reason to use `training-on-device` as the file name?

Reply:
It's fine :). I mean, this will make training-on-device.md and training-pytorch.md stay close when since the filenames are listed in alphabet order.

Reply:
Updated the name of the file. :)